### ISSUE TITLE
How to disable temporarily vcpkg

### ISSUE THREAD

**Issue by brupelo** [Microsoft/vcpkg#879]
Is it possible to disable vcpkg in one Visual Studio session and the enabling it back in other one?

The reason why I want to know this is because sometimes i use an [exe compressor](http://www.crinkler.net/) which replaces vs's linker but this one doesn't play well with vcpkg. Of course, If i rename the vcpkg folder for not being in PATH anymore this exe compressor will work but I was wondering if there was a more elegant way to overcome this little issue.

Thanks in advance.

**Comment by alexkaratarakis** [comment]
1) We don't currently have a per-VisualStudio-instance integration scheme, but you can toggle vcpkg's global integration on and off (respectively) with:
`vcpkg integrate install`
`vcpkg integrate remove`

2) Alternatively, there is a per-project integration with `vcpkg integrate project`; we mostly have this for cases where you have multiple `vcpkg` instances, but you can use it here as well. If you have two projects (or two copies of the same project), you can apply the integration to only one of the projects. Remember to also do `vcpkg integrate remove` to remove global integration.

(I think option 1 directly addresses your issue, but I pointed out 2 just in case)

**Comment by brupelo** [comment]
@alexkaratarakis Thanks, I think that will do it! Btw, one fast question, I've installed freeglut `vcpkg install freeglut` and now there are the proper include libraries `glut.h` and `freeglut.h` living in `vcpkg\installed\x86-windows\include\GL`. Here's the thing, let's say i've started sublime_text from the vs x86 command prompt and I got available both `cl` & `link`, If i try to compile a simple hello world like these ones inside sublime:

    #include <GL/glut.h>

    void displayMe(void) {
        glClear(GL_COLOR_BUFFER_BIT);
        glBegin(GL_POLYGON);
        glVertex3f(0.0, 0.0, 0.0);
        glVertex3f(0.5, 0.0, 0.0);
        glVertex3f(0.5, 0.5, 0.0);
        glVertex3f(0.0, 0.5, 0.0);
        glEnd();
        glFlush();
    }

    int main(int argc, char **argv) {
        glutInit(&argc, argv);
        glutInitDisplayMode(GLUT_SINGLE);
        glutInitWindowSize(300, 300);
        glutInitWindowPosition(100, 100);
        glutCreateWindow("Hello world :D");
        glutDisplayFunc(displayMe);
        glutMainLoop();
        return 0;
    }

I'll get this error `fatal error C1083: Cannot open include file: 'GL/glut.h': No such file or directory` , is there any way to overcome this issue and using vcpkg from sublime text (without using cmake, just using cl&link)?

**Comment by ras0219-msft** [comment]
Yep, it is just a bit less ergonomic.

Take a look at https://github.com/Microsoft/vcpkg/blob/master/docs/EXAMPLES.md#example-1-2-c, which lists the various folders of interest.

**Comment by brupelo** [comment]
@ras0219-msft Yeah, awesome, I got it working, thanks guys!

**Comment by FTamas77** [comment]
My Visual Studio 2022 says:

Severity	Code	Description	Project	File	Line	Suppression State
Error		CMake Error at sourceApp/CMakeLists.txt:1 (add_executable):
  Cannot find source file:

    sourceCode.cpp

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .mpp .m .M .mm .ixx .cppm .h
  .hh .h++ .hm .hpp .hxx .in .txx .f .F .for .f77 .f90 .f95 .f03 .hip .ispc		D:\Devel\Demo\sourceApp/CMakeLists.txt	1	


But I don't use it. I started to use it, but changed a little bit later. How can I at least temporally turn off?
